### PR TITLE
F.prototype

### PR DESCRIPTION
**Pull request by Amirhossein-Veysi**

### CODE REVIEW

**Comment by Amirhossein-Veysi** [comment]
Hello @mahdyar  and thanks for review,
/done

File: 1-js/08-prototypes/02-function-prototype/1-changing-prototype/task.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-importance: 5
+اهمیت: 5
```
Comment (mahdyar) [inline 1-js/08-prototypes/02-function-prototype/1-changing-prototype/task.md:1]: You shouldn't translate Metadata. [Read more](https://github.com/javascript-tutorial/fa.javascript.info#metadata)

**Review by mahdyar — CHANGES_REQUESTED** [review]

**Review by mahdyar — APPROVED** [review]

**Comment by mahdyar** [comment]
Thank you, @Amirhossein-Veysi!

**Pull request merged**